.github/workflows/c-cpp.yml

```
@@ -103,56 +109,65 @@
-        ./dbs26 -o- | sha1sum
-        ./dbs26
-        sha1sum dbs26.bin
-        rm -f dbs26.bin
+        "$__exe" -o- | sha1sum
+        "$__exe" -o "$RUNNER_TEMP/test.bin"
+        sha1sum "$RUNNER_TEMP/test.bin"
+        rm -f "$RUNNER_TEMP/test.bin"
 
     - name: Verify Windows binary output
+      env:
+        __exe: WINEDEBUG=-all wine64 ${{ github.env.__exe }}
```

actionlint (1.7.4)

receiver of object dereference "__exe" must be type of object but got "string"

(expression)